scripts/commit-with-submodule-log

```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
-
-$(dirname $0)/submodule-log "$@" | git commit --file -
```

Comment:
What is the purpose for deleting this file? As long as we continue to have submodules in diego-release, it seems like this is worth keeping.

Reply:
Ok, makes sense.